web-console/knikubevirt/vm-list/vm-list.md

```diff
@@ -0,0 +1,85 @@
+# VM List
+
+![List of VMs](img/0-0-main.png)
+
+The VM list contains columns for the following:
+- Name
+- Namespace
+- Status
+  - Running, Off, Error (with custom messages), Cloning, Migrating, Importing, Warning, Pending (with changes that will apply after restart), Unknown
+- Created
+  - Date formatting should follow OpenShift's console, viewable [here](https://github.com/openshift/console/blob/master/frontend/public/components/utils/timestamp.jsx) and [here](https://github.com/openshift/console/blob/master/frontend/public/components/utils/datetime.ts)
+  - If possible, a tooltip should appear on hover with the exact creation date, including year.
+- Node
+- IP Address
+- FQDN
+
+## Status tooltips
+
+![VM list status tooltips](img/0-0a-tooltips.png)
+
+Statuses other than Running or Off should be colored blue to indicate that they can be hovered and/or clicked.
```

Comment:
This is not consistent with OpenShift today. Definitely a cool idea, but I think we would want to change our status pattern across the board and not only for VMs! We should consider that change in a separate PR.